src/main/java/duke/ParsedAnswerHandler.java

```diff
@@ -0,0 +1,118 @@
+package duke;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * This class receives a ParsedAnswer object and execute commands based on information
+ * from the object.
+ */
+
+public class ParsedAnswerHandler {
+    ParsedAnswer pa;
+
+    ParsedAnswerHandler(ParsedAnswer pa) {
+        this.pa = pa;
+    }
+
+    void execute() {
+        String filePath = System.getProperty("user.dir")  + "/data/storage.txt";
+        switch (pa.getCommand()) {
+            case "bye":
+                System.out.println("Have a nice day.");
+                System.exit(0);
+                break;
+
+            case "list":
+                TaskList.list();
+                break;
+
+            case "todo":
+                ToDos td = new ToDos(pa.getDesc());
+                TaskList.add(td);
+                try {
+                    String sb = "T," +
+                            "1," +
+                            td.getDescription() + '\n';
```

Comment:
can consider line breaking such that the code is `+ td.getDescriptions() + '\n' `for formatting

Reply:
Agree. Moreover, since the codes inside each switch case are basically to execute something and to print something, you can abstract these processes in a new class.